pkg/cluster/node.go

```diff
@@ -0,0 +1,199 @@
+package cluster
```

Comment:
@jasonBirchall , can you order the functions so they are used in different purposes. Some are with receivers with Global((c *Cluster) HealthCheck()) and locals((c *Cluster) areNodesReady()), some global and local functions with Parameters. Maybe it makes sense if you are tying the recycle package.

Reply:
There are private and public functions/methods in this package. Are you suggesting grouping them together in separate files?

Reply:
Changed the name for readability.